src/korma/core.clj

```diff
@@ -533,7 +533,8 @@
 (defn exec
   "Execute a query map and return the results."
   [query]
-  (if (or (seq (:pre-side-effects query)) (seq (:post-side-effects query))
+  (if (or (seq (get-in query [:ent :pre-side-effects]))
```

Comment:
Hmm, was definitely incorrect before but I'm surprised it wasn't causing an obvious problem... thanks for catching that.

Reply:
You're right, with this fix it is now causing issues.